docs/plasma/index.rst

```diff
@@ -7,11 +7,20 @@ Plasma
 Introduction
 ============
 
-The :class:`Plasma` class is a basic structure to contain physical
-information about a plasma.  To initialize a plasma system, first
-create an instance of the :class:`Plasma` class and then set the
-:attr:`Plasma.density`, :attr:`plasma.momentum`,
-:attr:`plasma.pressure` and the :attr:`plasma.magnetic_field`.
+The :class:`Plasma3D` class is a basic structure to contain spatial
+information about a plasma.  To initialize a Plasma3D system, first
+create an instance of the :class:`Plasma3D` class and then set the
+:attr:`Plasma3D.density`, :attr:`Plasma3D.momentum`,
+:attr:`Plasma3D.pressure` and the :attr:`Plasma3D.magnetic_field`.
+
+This feature is currently under development.
+
+The :class:`PlasmaBlob` class is a basic structure to contain spatial
+information about a plasma.  To initialize a PlasmaBlob system, call
```

Comment:
I don't think `Blob` does any spatial information, does it? I though it was supposed to be just "plasma of any size  compressed into floats (okay, Quantities) as we don't care about spatial dependences in there".

Reply:
Fixed. Good catch!